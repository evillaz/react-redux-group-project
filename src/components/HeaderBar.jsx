import { NavLink } from 'react-router-dom';
import spaceLogo from './Images/planet.png';
import '../styles/style.css';

const HeaderBar = () => (
  <div className="header-bar">
    <NavLink to="/" style={{ color: '#000' }}>
      <div className="logo">
        <img src={spaceLogo} alt="Space Logo" />
        <h1>Space Travelers Hub</h1>
      </div>
    </NavLink>
    <nav className="nav-items">
      <ul>
        <li className="list-item"><NavLink to="/" style={{ color: '#000' }}>Rockets</NavLink></li>
        <li className="list-item"><NavLink to="missions" style={{ color: '#000' }}>Missions</NavLink></li>
        <li className="list-item"><NavLink to="profile" style={{ color: '#000' }}>My Profile</NavLink></li>
      </ul>
    </nav>
  </div>
);

export default HeaderBar;
