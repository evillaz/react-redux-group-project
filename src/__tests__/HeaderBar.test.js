import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import HeaderBar from '../components/HeaderBar';

describe('HeaderBar', () => {
  it('Testing of HeaderBar.jsx', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <HeaderBar />
      </MemoryRouter>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
