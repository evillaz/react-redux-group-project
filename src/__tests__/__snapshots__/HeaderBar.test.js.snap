// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['HeaderBar Testing of HeaderBar.jsx 1'] = `
<DocumentFragment>
  <div
    class="header-bar"
  >
    <a
      aria-current="page"
      class="active"
      href="/"
      style="color: rgb(0, 0, 0);"
    >
      <div
        class="logo"
      >
        <img
          alt="Space Logo"
          src="planet.png"
        />
        <h1>
          Space Travelers Hub
        </h1>
      </div>
    </a>
    <nav
      class="nav-items"
    >
      <ul>
        <li
          class="list-item"
        >
          <a
            aria-current="page"
            class="active"
            href="/"
            style="color: rgb(0, 0, 0);"
          >
            Rockets
          </a>
        </li>
        <li
          class="list-item"
        >
          <a
            class=""
            href="/missions"
            style="color: rgb(0, 0, 0);"
          >
            Missions
          </a>
        </li>
        <li
          class="list-item"
        >
          <a
            class=""
            href="/profile"
            style="color: rgb(0, 0, 0);"
          >
            My Profile
          </a>
        </li>
      </ul>
    </nav>
  </div>
</DocumentFragment>
`;
